docs/src/architectures/pet.rst

```diff
@@ -161,6 +161,98 @@ result in a model that is about 1.5 times more lightweight and faster, with an
 expected minimal deterioration in accuracy.
 
 
+Description of Hyperparameters
+------------------------------
+
+``RANDOM_SEED``: random seed
+``CUDA_DETERMINISTIC``: if applying PyTorch reproducibility settings
+``MULTI_GPU``: use multi-GPU training (on one node) using DataParallel from 
+PyTorch-Geometric
+
+``MODEL_TO_START_WITH``: If not None, then training will be continued from this 
```

Comment:
Is this possible to use through metatrain?

Reply:
Probably not, I just deleted it